core/src/main/java/org/elasticsearch/index/query/FQueryFilterBuilder.java

```diff
@@ -109,28 +86,25 @@ public QueryValidationException validate() {
     }
 
     @Override
-    public int hashCode() {
+    public int doHashCode() {
```

Comment:
I found a couple of other places where I have the same leftovers, going to clean those up